buildrunner/steprunner/tasks/build.py

```diff
@@ -31,51 +32,28 @@ class BuildBuildStepRunnerTask(BuildStepRunnerTask):  # pylint: disable=too-many
     def __init__(
         self,
         step_runner,
-        config,
+        step: StepBuild,
```

Comment:
Should we also add typing for the other arguments?

Reply:
Same answer as above